charts/testkube-cloud-api/values.yaml

```diff
@@ -60,6 +60,26 @@ global:
   nats:
     # -- NATS URI
     uri: ""
+  ## Global S3 configuration for Testkube Pro
+  storage:
+    # -- S3 bucket in which Test Artifacts & Logs will be stored
+    outputsBucket: ""
+    # -- Endpoint to a S3 compatible storage service (without protocol)
+    endpoint: ""
+    # -- S3 region
+    region: ""
+    # -- Toggle whether to use HTTPS when connecting to the S3 server
+    secure: null
+    # -- Toggle whether to skip verifying TLS certificates
+    skipVerify: null
+    # -- S3 Access Key ID
+    accessKeyId: ""
+    # -- S3 Secret Access Key
+    secretAccessKey: ""
+    # -- S3 Token
+    token: ""
+    # -- Credentials secret ref (secret should contain keys: root-user, root-password, token) (default is `testkube-cloud-minio-secret`)
```

Comment:
Can we make this default `testkube-minio-credentials` somehow? I think using the word cloud or enterprise are confusing for On-Prem users.

Reply:
It would be done in a follow-up PR